### PR TITLE
feat/fix: RegularDecisionTree

### PR DESCRIPTION
**Pull request by younesdessia**
- feat: add visualize_tree method
- fix: RegularDecisionTree

### CODE REVIEW

File: dectree/dectree.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 from operator import itemgetter
 import numpy as npy
 import matplotlib.pyplot as plt
+import graphviz
```
Comment (masfaraud) [inline dectree/dectree.py]: Not a fan of graphviz import. Is it necessary?

**Review by masfaraud — CHANGES_REQUESTED** [review]

File: dectree/dectree.py (hunk 1 above)
Comment (younesdessia) [reply dectree/dectree.py]: @masfaraud 
No, I removed it !

File: setup.py [hunk 2]
```diff
@@ -64,4 +64,4 @@ def get_version():
       author_email='root@dessia.tech',
       packages=['dectree'],
       install_requires=['numpy',
-                        'matplotlib'])
+                        'matplotlib', 'graphviz'])
```
Comment (AxelRinghausen) [inline setup.py]:
```suggestion
                        'matplotlib'])
```

**Review by AxelRinghausen — CHANGES_REQUESTED** [review]

**Comment by younesdessia** [comment]
@AxelRinghausen 
ready for review and merge

**Review by masfaraud — APPROVED** [review]

**Review by AxelRinghausen — APPROVED** [review]

**Pull request merged**